firmware/library/L1_Drivers/LCD_I2C_Backpack/I2CBackpack.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <cstdint>
+
+// Import SJOne libraries to be used for testing
+#include "L2_Drivers/base/i2c_base.hpp"
+
+class BackpackInterface
```

Comment:
This title isn't very informative. A backpack could be many different things. You may want to say LcdBackpackIntface. The same interface could be used for a UART backpack, but probably not for a backpack meant to control a pwm generation device.